src/cmd/linuxkit/run_vcenter.go

```diff
@@ -90,7 +92,7 @@ func runVcenter(args []string) {
 	}
 
 	// Test any passed in files before creating a new VM
-	checkFile(*newVM.path)
+	// checkFile(*newVM.path)
```

Comment:
thanks @riyazdf, then I can delete those two functions from `runVcenter`. I will create a separate commit for it.